iceoryx_binding_c/test/moduletests/main.cpp

```diff
@@ -27,10 +27,10 @@ using ::testing::_;
 
 // in case this isn't executed before the other tests, just call
 // 'checkIceoryxBindingCStorageSizes' directly in main
-/// @todo iox-#308 enable this check once the stack is used to store the objects
-TEST(SanityCheck, DISABLED_CheckStorageSizeAndAlingment)
+TEST(SanityCheck, CheckStorageSizeAndAlingment)
 {
     ::testing::Test::RecordProperty("TEST_ID", "879abe49-6cfb-43f8-a9a1-b6c20b99293f");
+    GTEST_SKIP() << "Enable this check once the stack is used to store the objects iox-#308";
```

Comment:
You did not remove the todo from the other files. Was this intentional?

Another question. What do you think about this format `GTEST_SKIP() << "todo iox-#0815 Bla bla`? We might reduce duplication.

Reply:
But we could also remove the todo. In the end we never would like to skip tests, so what about the simple format `GTEST_SKIP() << "iox-#1234 blalall"`

And since every rule needs an exception, the exception here would be if we disable tests for a certain platform. 
`GTEST_SKIP() << "not supported for windows"` with an `ifdef`. But here we could also introduce a small function `SUPPORTED_PLATFORMS(__WIN32, __LINUX__)` or `SKIP_FOR_PLATFORM(__QNX__)`.

Reply:
The `todo` indicates here that there is some follow-up work still left and should be tracked. Otherwise you are forced to add a doxygen todo near to the test.